Ignore nonexistance of agetty

Vogtinator opened the pull request:
Fixes this error during installation of *-release:

    /usr/sbin/issue-generator: line 111: /usr/sbin/agetty: No such file or directory

thkukuk commented:
Shouldn't this better be "test -x /usr/sbin/agetty && ..." ?

Vogtinator commented:
Could be, but any issue with `agetty` can be safely ignored as it's just a harmless reload.
If you think a test is better, I can do that as well.

thkukuk commented:
On Wed, Feb 07, Fabian Vogt wrote:

> Could be, but any issue with agetty can be safely ignored as it's just a
> harmless reload.
> If you think a test is better, I can do that as well.

I think we should add the test additional, it's cleaner.

-- 
Thorsten Kukuk, Distinguished Engineer, Senior Architect SLES & CaaSP
SUSE LINUX GmbH, Maxfeldstr. 5, 90409 Nuernberg, Germany
GF: Felix Imendoerffer, Jane Smithard, Graham Norton, HRB 21284 (AG Nuernberg)

Vogtinator committed:
Ignore nonexistance of agetty

Fixes this error during installation of *-release:

    /usr/sbin/issue-generator: line 111: /usr/sbin/agetty: No such file or directory

Vogtinator commented:
Done.
